### PR TITLE
Electron-249 (Bring Wrapper to Front)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
## Description
A new feature/option that brings electron wrapper to front when a user receives a notification [JIRA-ticket](https://perzoinc.atlassian.net/browse/ELECTRON-249)

screen shot 2018-01-17 at 3 28 03 pm

@VikasShashidhar & @VishwasShashidhar Please review. Thanks :)

### CODE REVIEW

File: js/menus/menuTemplate.js [hunk 1]
```diff
@@ -254,6 +255,17 @@ function getTemplate(app) {
         }
     });
 
+    // Window menu -> bringToFront
```
Comment (VikasShashidhar) [inline js/menus/menuTemplate.js]: We might have to go with a different name for this. There is already an option called "Bring all to front" in the Window menu. Let's ask product about this.

File: js/menus/menuTemplate.js [hunk 2]
```diff
@@ -284,7 +296,7 @@ function setCheckboxValues() {
         /**
          * Method that reads multiple config fields
          */
-        getMultipleConfigField(['minimizeOnClose', 'launchOnStartup', 'alwaysOnTop', 'notificationSettings'])
+        getMultipleConfigField(['minimizeOnClose', 'launchOnStartup', 'alwaysOnTop', 'notificationSettings', 'bringToFront'])
```
Comment (VikasShashidhar) [inline js/menus/menuTemplate.js]: I would move these out to constants now that they are growing.

File: js/menus/menuTemplate.js (hunk 1 above)
Comment (VikasShashidhar) [reply js/menus/menuTemplate.js]: @KiranNiranjan Please change the verbatim to "Bring to Front on Notifications" in all the necessary places.

Comment (KiranNiranjan) [reply js/menus/menuTemplate.js]: Done 👍

File: js/mainApiMgr.js [hunk 3]
```diff
@@ -82,6 +83,11 @@ electron.ipcMain.on(apiName, (event, arg) => {
     }
 
     if (arg.cmd === apiCmds.activate && typeof arg.windowName === 'string') {
```
Comment (VikasShashidhar) [inline js/mainApiMgr.js]: Not good having all these ```if``` blocks. Should probably switch to a switch-case block (no pun).

Comment (KiranNiranjan) [reply js/mainApiMgr.js]: Update in this PR -> https://github.com/symphonyoss/SymphonyElectron/pull/292

**Comment by VikasShashidhar** [comment]
Please resolve merge conflicts

**Pull request merged**